docs/man/pg_activity.1.old

```diff
@@ -0,0 +1,370 @@
+.\" Automatically generated by Pod::Man 4.10 (Pod::Simple 3.35)
```

Comment:
This file should not be committed I guess.

Reply:
Fixed